framework/components/ema-content-slider/ema-content-slider.scss

```diff
@@ -0,0 +1,45 @@
+.ema-content-slider {
+  margin: 0;
+}
+
+.ema-content-slider .ecl-list-item.ema-content-slider__list-item--default {
+  border: 2px solid transparent;
+  padding: map-get($ecl-spacing, 'xs');
+}
+
+.ema-content-slider .ecl-list-item.ema-content-slider__list-item--active {
+  border: 2px solid map-get($ema-colors, 'blue-3');
+}
+
+.ema-content-slider .ecl-list-item.ema-content-slider__list-item--highlight {
```

Comment:
Should we agree with class concatenations like this? I had to do it because `ecl-meta` by design takes the same and all the classes from its parent `ecl-list-item`. Opinions?

Reply:
I don't get it. If you use just `.ema-content-slider__list-item--highlight` it should work.

Reply:
not really. do a `document.querySelectorAll('.ema-content-slider__list-item--highlight')` and you will see more results than wanted. We are investigating the reason why children inherit parent' `extra_classes`